### PR TITLE
CJsonObejct harness

### PR DESCRIPTION
**Pull request by baptistapedro**
Seems that CJsonObject is forked in Mayhemheroes but no harness has been added yet.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM fuzzers/afl:2.52
```
Comment (xansec) [inline Dockerfile]: Can we please add a package stage here?

File: fuzzers/fuzz1.cpp [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#include <string>
```
Comment (xansec) [inline fuzzers/fuzz1.cpp]: Can we rename this to indicate what's being fuzzed? For example, "fuzz_cjson_to_string.cpp" or similar.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Review by baptistapedro — COMMENTED** [review]
Changed the harness name to indicate what is being fuzzed

File: Dockerfile (hunk 1 above)
Comment (baptistapedro) [reply Dockerfile]: I can refactor the dockerfile. I assume you mean to use a distro package instead right, like FROM/ubuntu:20.04
Will do it shortly..

Comment (xansec) [reply Dockerfile]: No - I mean can we separate the Dockerfile into a build and package stage. Here's an example of another PR with a package stage:
https://github.com/evanrichter/watchman/blob/main/mayhem/Dockerfile
See here for more info: https://docs.docker.com/build/building/multi-stage/

Comment (baptistapedro) [reply Dockerfile]: Done

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,20 @@
+FROM fuzzers/afl:2.52 as builder
```
Comment (xansec) [inline Dockerfile:1]: Awesome!

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**